src/mkindex.hpp

```diff
@@ -57,6 +58,12 @@ template <DbIndexType   c_indexType,
           AlphabetEnum  c_transAlph>
 void argConv3(LambdaIndexerOptions const & options);
 
+template <DbIndexType   c_indexType,
+          AlphabetEnum  c_origAlph,
+          AlphabetEnum  c_transAlph>
+void argConv4(LambdaIndexerOptions const & options);
```

Comment:
Maybe rename these to `argConv3a` and `argConv3b` to indicate that they are called alternatively and not in succession?